thruster_control/nodes/fault_thrust_distributor_node.py

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env python
```

Comment:
rename file to faulty_thrust_distributor_node.py for consistency, along with changing it in the other places. You could also use noisy instead of faulty, just as long as its consistent.